apps/openmw/mwgui/windowmanagerimp.cpp

```diff
@@ -566,10 +566,10 @@ namespace MWGui
 
                     // Show the windows we want
                     mMap            ->setVisible(eff & GW_Map);
```

Comment:
Funny that this line isn't causing a warning. But can we change it for consistency anyway? Looks a bit strange like this.